sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/uninstall/TaskKillStep.java

```diff
@@ -0,0 +1,36 @@
+package com.mesosphere.sdk.scheduler.uninstall;
+
+import com.mesosphere.sdk.scheduler.TaskKiller;
+import com.mesosphere.sdk.scheduler.plan.PodInstanceRequirement;
+import com.mesosphere.sdk.scheduler.plan.Status;
+import com.mesosphere.sdk.scheduler.recovery.RecoveryType;
+import org.apache.mesos.Protos;
+
+import java.util.Optional;
+
+/**
+ * Step which issues kill commands to all .
+ */
+public class TaskKillStep extends UninstallStep {
+
+    private TaskKiller taskKiller;
+    private Protos.TaskID taskID;
+
+    public TaskKillStep(Status status, Protos.TaskID taskID) {
```

Comment:
Given that the passed status is always PENDING, could skip the param and just set that internally